SharePDF/SharePDF/README.md

```diff
@@ -0,0 +1,67 @@
+# How to share the PDF document in .NET MAUI PDF Viewer?
+
+This project demonstrates how to share a PDF document using .NET MAUI and Syncfusion PDF Viewer. The application loads a PDF file and enables users to share it through a button click.
+
+# Features
+1. Load a PDF in the Syncfusion PDF Viewer.
+2. Save the PDF document to a local file.
+3. Share the PDF using the mobile platform’s share functionality.
+
+# Prerequisites
+1. .NET MAUI installed.
+2. Syncfusion.Maui.PdfViewer NuGet package.
+
+# Steps
+## 1. Set Up the PDF Viewer
+In your MainPage.xaml.cs, the PDF Viewer is initialized with a PDF document embedded in your resources.
+
+```csharp
+pdfViewer.DocumentSource = this.GetType().Assembly.GetManifestResourceStream("SharePDF.Assets.PDF_Succinctly.pdf");
+```
+## 2. Share PDF Functionality
```

Comment:
Are you sharing any functionality?

<img width="273" alt="image" src="https://github.com/user-attachments/assets/615dcd4d-02b6-4d44-a5d6-b60fb839b426" />